src/Components/ArticlesPage.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState, useEffect } from 'react';
+import axios from 'axios';
+import ArticlesList from './ArticlesList'
+
+function ArticlesPage() {
+    const [articles, setArticles] = useState([])
+    const [loading, setLoading] = useState(true)
+    const [error, setError] = useState(null)
+
+    useEffect(() => {
+        const fetchArticles = async () => {
+            try {
+               
+               const response = await axios.get('https://sams-news.onrender.com/api/articles')
+            
+               setArticles(response.data.articles)
+              
+                setLoading(false)
+            } catch (err) {
+                
+                setError(err.message)
+                setLoading(false)
+            }
+        }
```

Comment:
This does work, but would be a lot cleaner if your api functions are in a separate file. Personal preference though